src/components/saas-list/saas-list.controller.ts

```diff
@@ -11,21 +11,19 @@ export class RHListController implements angular.IController {
       showSelectBox: false,
       onClick: this.handleClick
     };
-    this.ctrl.loading = true;
   }
 
-  public $onInit() {
-    this.ctrl.loading = _.isEmpty(this.ctrl.applications);
+  public hasSaasOfferings (): boolean {
+    return !_.isEmpty(this.ctrl.saasOfferings);
   }
 
   public handleClick(item: any, e: any) {
     window.open(item.url, '_blank');
   };
 
   public $onChanges(onChangesObj: angular.IOnChangesObject) {
-    if ((onChangesObj.applications && !onChangesObj.applications.isFirstChange()) ) {
-      this.ctrl.applications = onChangesObj.applications.currentValue;
-      this.ctrl.loading = false;
+    if ((onChangesObj.saasOfferings && !onChangesObj.saasOfferings.isFirstChange()) ) {
+      this.ctrl.saasOfferings = onChangesObj.saasOfferings.currentValue;
     }
```

Comment:
$onChanges is triggered when the parent changes the value of a one-way binding.  $onChanges doesn't get called when the variable is changed from within the component.   $onChanges probably isn't needed at this point since we're setting the saas offerings synchronously from the constants file.  I left it in because there is talk that in the future they might be loaded via an asynchronous API.